lambdas/README.md

```diff
@@ -1,12 +1,64 @@
 # Lambdas
 
-This lambda is to update the view dynamo table from the main one.
+These lambdas relate to automated tasks, run on a daily or weekly basis, which copy promotions data stored in the DynamoDB tables over to SalesForce and BigQuery, as well as backing up the DB tables themselves.
+
+Five lambda functions are defined in the lambdas/src folder. They relate to eight lambdas in AWS,  in the memsub-promotions-lambdas application - link here: https://eu-west-1.console.aws.amazon.com/lambda/home?region=eu-west-1#/applications/memsub-promotions-lambdas 
+
+We can also list just the Lambdas in AWS using this link: https://eu-west-1.console.aws.amazon.com/lambda/home?region=eu-west-1#/functions?fo=and&k0=functionName&o0=%3A&v0=MembershipSub
+
+We can view the DynamoDB tables at this link (search on ‘prom’): https://eu-west-1.console.aws.amazon.com/dynamodbv2/home?region=eu-west-1#tables 
+
+
+### MembershipSub-PromoCode-View-Dynamo-to-Data-Lake
+A Lambda function to backup the PromoCode-View Dynamo table to the Data Lake.
+
+The code dumps data to a csv in S3 https://github.com/guardian/memsub-promotions/blob/main/lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Data-Lake.js#L16
+It runs on a daily schedule; for dev work we can run it manually and check that the csv gets updated. Always download the csv file from S3 before doing any dev/testing work - that way if anything does go wrong we can revert the code and reload the original file back into S3 to minimise damage.
+
+A single version (PROD) of this lambda exists in AWS, and will rebuild automatically when a PR branch merges into `main`.
+
+### MembershipSub-PromoCode-View-Dynamo-to-Salesforce
+A Lambda function to replicate the MembershipSub-PromoCode-View-DEV Dynamo table to Salesforce.
+
+This lambda sends data to salesforce on a daily schedule. We can just check the lambda's cloudwatch logs to confirm that it's happy - it logs all the requests it's making.
+
+Three versions of this lambda (PROD, UAT, DEV) exist in AWS, each of which maps to thge equivalent environment in the SalesForce cloud. We can use the DEV version for testing. 
+
+The lambda relies on SalesForce usernames/passwords which have been stored in AWS in ???
```

Comment:
@tomrf1 - do you want to add additional details here?

Reply:
They're configured as encrypted environment variables

Reply:
Added and pushed. Thanks!